kernel/kernel-api/src/main/java/io/delta/kernel/utils/DataFileStatus.java

```diff
@@ -34,11 +36,17 @@ public class DataFileStatus extends FileStatus {
    * @param size File size in bytes.
    * @param modificationTime Last modification time of the file in epoch milliseconds.
    * @param statistics Optional column and file level statistics in the data file.
+   * @param dvDescriptor Optional Deletion Vector descriptor in the data file.
    */
   public DataFileStatus(
-      String path, long size, long modificationTime, Optional<DataFileStatistics> statistics) {
+      String path,
+      long size,
+      long modificationTime,
+      Optional<DataFileStatistics> statistics,
+      Optional<DeletionVectorDescriptor> dvDescriptor) {
```

Comment:
`DeletionVectorDescriptor` is an internal class and it is not part of the public API